lib/diaspora/shareable.rb

```diff
@@ -50,6 +50,14 @@ def subscribers
       end
     end
 
+    # Remote pods which are known to be subscribed to the post. Must include all pods which received the post in the
+    # past.
+    #
+    # @return [Array<String>] The list of pods' URIs
+    def subscribed_pods_uris
+      subscribers.map {|subscriber| subscriber.pod.try(:url_to, "") }.compact.uniq
```

Comment:
This will not work for public posts since for public posts `subscribers` return an array of people, not a relation.